tests/model_registry/test_model_registry_creation.py

```diff
@@ -0,0 +1,41 @@
+import shlex
+from ast import literal_eval
+from typing import Self
+from simple_logger.logger import get_logger
+from ocp_resources.model_registry import ModelRegistry
+from pyhelper_utils.shell import run_command
+
+from tests.model_registry.utils import generate_register_model_command
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestModelRegistryCreation:
+    """Tests the creation of a model registry"""
+
+    # TODO: Enable Model Registry in DSC if needed
+
+    def test_model_registry_instance_creation(self: Self, model_registry_instance: ModelRegistry):
+        assert model_registry_instance.exists
+
+    def test_registering_model(self: Self, model_registry_instance_rest_endpoint: str, current_client_token: str):
+        cmd = generate_register_model_command(
+            endpoint=model_registry_instance_rest_endpoint, token=current_client_token
+        )
+        _, out, _ = run_command(command=shlex.split(cmd))
+        out_dict = literal_eval(out)
+        errors = []
+        if not out_dict["name"] == "model-name":
```

Comment:
will you always have these keys or would it be better to use `get`?

Reply:
Unless the API changes the response will remain consistent